autospotting.go

```diff
@@ -58,7 +59,11 @@ func run() {
 		conf.TagFilteringMode,
 		conf.SpotProductDescription)
 
-	autospotting.Run(conf.Config)
+	xray.Configure(xray.Config{
+		ContextMissingStrategy: ctxmissing.NewDefaultLogErrorStrategy(),
+		LogLevel:               "error",
+	})
```

Comment:
What exactly do you want to configure in this place? Log level?

Reply:
I made log level configurable via flag, but it looks like there is a bug in X-Ray SDK. https://github.com/aws/aws-xray-sdk-go/issues/60. At the moment, it will post a particular error message ignoring log level...